redash/query_runner/mixpanel.py

```diff
@@ -0,0 +1,198 @@
+import json
+import urllib2
+from redash.query_runner import *
+from collections import Iterable
+
+try:
+    from mixpanel_api import Mixpanel
+    from ssl import SSLError
+    import urllib2
+    import base64
+except ImportError:
+    enabled = False
+else:
+    enabled = True
+
+class MixpanelPatched(Mixpanel):
```

Comment:
Can you please add a comment explaining why this is needed?

Reply:
Also, this should be placed in the import try/except block, because otherwise `Mixpanel` might be not defined.